Fix test that fails on i686:

https://bitbucket.org/cffi/cffi/issues/382

This is a squashed version of these commits:
https://bitbucket.org/cffi/cffi/commits/ef09637b2314
https://bitbucket.org/cffi/cffi/commits/7a76a3815340

diff -r 99940f1f5402 testing/cffi0/test_function.py
--- a/testing/cffi0/test_function.py	Tue Feb 27 21:18:33 2018 +0100
+++ b/testing/cffi0/test_function.py	Sun Nov 11 16:26:23 2018 +0100
@@ -45,14 +45,14 @@
         assert x != math.sin(1.23)    # rounding effects
         assert abs(x - math.sin(1.23)) < 1E-6
 
-    def test_sin_no_return_value(self):
+    def test_lround_no_return_value(self):
         # check that 'void'-returning functions work too
         ffi = FFI(backend=self.Backend())
         ffi.cdef("""
-            void sin(double x);
+            void lround(double x);
         """)
         m = ffi.dlopen(lib_m)
-        x = m.sin(1.23)
+        x = m.lround(1.23)
         assert x is None
 
     def test_dlopen_filename(self):
